js/gated-content/src/mixins/EventMixin.js

```diff
@@ -36,22 +34,28 @@ export const EventMixin = {
       return this.$store.getters.getAppSettings;
     },
     date() {
-      return moment(this.video.attributes.date.value).format('dddd, MMMM Do, YYYY');
+      return dayjs(new Date(this.video.attributes.date.value)).format('dddd, MMMM Do, YYYY');
     },
     time() {
-      return moment(this.video.attributes.date.value).format('h:mm a');
+      return dayjs(new Date(this.video.attributes.date.value)).format('h:mm a');
     },
     duration() {
-      return moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment(this.video.attributes.date.end_value))).humanize();
+      const min = Math.floor(dayjs.duration(
+        new Date(this.video.attributes.date.end_value) - new Date(this.video.attributes.date.value),
+      ).asMinutes());
+
+      return `${min} ${this.$options.filters.simplePluralize('minute', min)}`;
     },
     startsIn() {
-      const duration = moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment()));
-      if (duration.asHours() > 48) {
-        return duration.format('d [day]');
+      const eventStartDate = new Date(this.video.attributes.date.value);
```

Comment:
can we do here this thing:
```suggestion
      const eventStartDate = dayjs(this.video.attributes.date.value);
```